lockup/src/lib.rs

```diff
@@ -926,53 +934,65 @@ mod tests {
             contract.get_liquid_owners_balance().0,
             to_yocto(1000) - MIN_BALANCE_FOR_STORAGE
         );
-        assert_eq!(contract.get_locked_vested_amount().0, to_yocto(0));
+        assert_eq!(contract.get_locked_vested_amount(no_vesting_schedule()).0, to_yocto(0));
         assert_eq!(contract.get_locked_amount().0, to_yocto(0));
-        assert_eq!(contract.get_unvested_amount().0, to_yocto(0));
+        assert_eq!(contract.get_unvested_amount(no_vesting_schedule()).0, to_yocto(0));
     }
 
     #[test]
     fn test_termination_no_staking() {
         let mut context = basic_context();
         testing_env!(context.clone());
-        let mut contract = new_contract(true, new_vesting_schedule(0), None, true);
+        let vesting_schedule = new_vesting_schedule(0);
+        let mut contract = LockupContract::new(
+            account_owner(),
+            to_nanos(YEAR).into(),
+            None,
+            TransfersInformation::TransfersEnabled {
+                transfers_timestamp: to_ts(GENESIS_TIME_IN_DAYS).into(),
+            },
+            VestingScheduleOrHash::VestingSchedule(vesting_schedule.clone()),
+            None,
+            AccountId::from("whitelist"),
+            Some(account_foundation()),
+        );
```

Comment:
Can you update `new_contract` with whatever changes needed to avoid initializing the contract here?